osu.Game/Overlays/Wiki/WikiArticlePage.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using Markdig.Syntax;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Game.Overlays.Wiki.Markdown;
+
+namespace osu.Game.Overlays.Wiki
+{
+    public class WikiArticlePage : GridContainer
```

Comment:
Should probably be a `CompositeDrawable` with the `GridContainer` as an `InternalChild`. Otherwise this is outwardly exposing `GridContainer` members, such as cell content setters, that shouldn't really be exposed.